tasks/config.yml

```diff
@@ -89,6 +89,32 @@
      dest: /etc/nginx/sites-available/bigbluebutton
   notify: reload nginx
 
+- name: disable CPUSchedulingPolicy in FreeSWITCH service
+  lineinfile:
+    path: /lib/systemd/system/freeswitch.service
+    regexp: '^CPUSchedulingPolicy=rr'
+    line: '#CPUSchedulingPolicy=rr'
+  notify: restart freeswitch
+  when: not bbb_cpuschedule
+
+- name: disable IPv6 in FreeSWITCH service 1/2
```

Comment:
this should also only happen `when: not bbb_freeswitch_ipv6`?

Reply:
@robbi5 yes, otherwise it should not disable IPv6 in FreeSWITCH service.

Do I miss anything?

Reply:
@robbi5 Thanks, I missed this.

Reply:
Is there a option to enable ipv6 again?

Reply:
> Is there a option to enable ipv6 again?

I would quite appreciate this option...

Reply:
I have now included support to enable IPv6 in FreeSWITCH service.